nf03_test4/run_f90_par_test.sh.in

```diff
@@ -11,7 +11,10 @@ echo "Testing netCDF parallel I/O through the F90 API."
 @MPIEXEC@ -n 4 ./f90tst_parallel3
 @MPIEXEC@ -n 8 ./f90tst_nc4_par
 @MPIEXEC@ -n 4 ./f90tst_parallel_fill
-@MPIEXEC@ -n 4 ./f90tst_parallel_compressed
+
+if test x@HAS_NC4_PARALLEL@ = xyes ; then
```

Comment:
There's no need to test this. If it is not yes, the directory would not be built, or the parallel tests would not be run. So it will always be "yes" here...